packages/respect-core/src/modules/__tests__/config-parser/resolve-reusable-component-item.test.ts

```diff
@@ -1,11 +1,16 @@
-import type { TestContext } from '../../../types.js';
+import { logger } from '@redocly/openapi-core';
+import { type TestContext } from '../../../types.js';
 
 import { resolveReusableComponentItem } from '../../context-parser/index.js';
 
 describe('resolveReusableComponentItem', () => {
   it('should return parameter if not reference', () => {
     expect(
-      resolveReusableComponentItem({ in: 'query', name: 'test', value: 'test' }, {} as any)
+      resolveReusableComponentItem({ in: 'query', name: 'test', value: 'test' }, {
+        options: {
+          logger,
+        },
+      } as unknown as TestContext)
```

Comment:
?

```suggestion
      } as Partial<TestContext> as TestContext)
```

Or what is the issue?